modules/oe_paragraphs_illustrations_lists/config/install/field.storage.paragraph.field_oe_illustration_ratio.yml

```diff
@@ -0,0 +1,20 @@
+langcode: en
+status: true
+dependencies:
+  module:
+    - options
+    - paragraphs
+id: paragraph.field_oe_illustration_ratio
+field_name: field_oe_illustration_ratio
+entity_type: paragraph
+type: list_string
+settings:
+  allowed_values: {  }
+  allowed_values_function: _oe_paragraphs_illustrations_lists_allowed_values_ratios
+module: options
+locked: false
+cardinality: 1
+translatable: true
```

Comment:
Shouldn't be translatable, as I understand.

Reply:
Ok